### PR TITLE
Add changes from externally changed version

### PR DESCRIPTION
**Pull request by wolfs**
This fixes some deprecation warnings. Moreover, `addSources` works
again.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,12 +1,13 @@
 buildscript {
     repositories {
         mavenLocal()
-        jcenter()
         google()
+        jcenter()
+		mavenCentral()
```
Comment (lptr) [inline build.gradle]: Should we add `.editorconfig`?

File: build.gradle [hunk 2]
```diff
@@ -63,11 +66,13 @@ subprojects { project ->
             }
             resolutionStrategy {
                 force 'com.google.guava:guava:19.0'
+                // Not in externally edited version
                 force 'com.google.auto:auto-common:0.6'
+                force 'io.reactivex:rxandroid:1.2.1'
```
Comment (lptr) [inline build.gradle:71]: Seems like we should be able to do these things more elegantly / in one place?

File: gradle.properties [hunk 3]
```diff
@@ -7,3 +7,10 @@ org.gradle.parallel=true
 org.gradle.configureondemand=false
 android.enableProfileJson=false
 org.gradle.jvmargs=-Xmx8G
+org.gradle.workers.max=16
+
+#android.enableNewDsl=false
+
+#android.enableDexingArtifactTransform=true
+#android.enableSeparateRClassCompilation=true
+#android.enableGradleWorkers=true
```
Comment (lptr) [inline gradle.properties]: We need `.editorconfig`.

File: build.gradle (hunk 2 above)
Comment (wolfs) [reply build.gradle:71]: Yes, we should. I'll ask the dependency management team for help what is the current way to solve this problem. I wouldn't do it as part of this PR, though.

File: build.gradle (hunk 1 above)
Comment (wolfs) [reply build.gradle]: Added.

**Comment by wolfs** [comment]
@lptr PTAL!

**Review by lptr — APPROVED** [review]

File: build.gradle [hunk 4]
```diff
@@ -174,6 +183,7 @@ task addSources {
                 println "Adding sources to: " + sourceFolder
                 copy {
                     from fileTree(new File(project.rootDir, "gradle/SourceTemplate/app/src/")).exclude("main/AndroidManifest.xml")
+                    // Not in externally edited version
```
Comment (oehme) [inline build.gradle:186]: What do all these comments mean?

**Pull request merged**